### PR TITLE
migrate the hail on dataproc examples...

### PR DESCRIPTION
**Pull request by amygdala**
... and make some updates & add additional text for sharing externally. 
I expect you may have some comments on how to best phrase some stuff I added, plus there may be some more spots that need additional explanation.

Some things to note:
- the links at the top of the notebooks (viewing in GH + importing to a nb vm) are for when this branch is merged into in `main`.  I'll double check them then to make sure I didn't make a mistake.
- There shouldn't be any code changes, only markdown cell changes (& description .md changes). However, this PR doesn't show a direct diff with the originals from the other repo (sorry, didn't think ahead)
- I will also create a prod workspace that they will _duplicate_ (this is the current terminology over 'clone' IIUC) to create their own. It will have the buckets created and the GH repo added. 

I haven't yet tested this in prod (dataproc enablement just landed today) but will shortly.

### CODE REVIEW

File: dataproc/hail_demo_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# How do I get started?
+
+*Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace. 
+
+
+## Step 1: Preview prior runs of the relevant notebooks.
+
+On the 'Resources' page, navigate to TVC folder "Notebook snapshots" and preview the notebooks you see there in this order to get a deeper understanding of what setup and analysis this demonstration workspace enables:
+1. `workspace_setup.ipynb` (This notebook was run as part of the "Hail Demo" workspace creation, and you do not need to run it again in your duplicated workspace).
```
Comment (deflaux) [inline dataproc/hail_demo_dashboard.md]: `workspace_setup.ipynb` no longer creates resources that are cloned by default, so people do need to run it once per workspace

please update this sentence and also add an extra step in the list of steps below

File: dataproc/create_hail_cluster.ipynb [hunk 2]
```diff
@@ -0,0 +1,865 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6376d1d7-1016-47b1-8f86-ecb021d147af",
+   "metadata": {},
+   "source": [
+    "## Check the Enterprise Terra reference to the Dataproc temp bucket\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note:</b> If you did not reach this notebook by duplicating the \"Hail Demo\" workspace, then you will need to first run  <a href=\"https://github.com/DataBiosphere/terra-axon-examples/blob/main/workspace_setup.ipynb\">workspace_setup.ipynb</a>, which is in the parent directory of this notebook.  \n",
```
Comment (deflaux) [inline dataproc/create_hail_cluster.ipynb]: `workspace_setup.ipynb` no longer creates resources that are cloned by default, so people do need to run it once per workspace

File: dataproc/create_hail_cluster.ipynb [hunk 3]
```diff
@@ -0,0 +1,865 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1e9af1d8-64f4-4d1a-ba32-e5eb4eff6565",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!terra resource resolve --name dataproc_staging_{USER} || terra resource create gcs-bucket --name=dataproc_staging_{USER} \\\n",
```
Comment (deflaux) [inline dataproc/create_hail_cluster.ipynb]: nit: line is a bit too long, consider adding slash-newline so the bash code displays well on GitHub

File: dataproc/create_hail_cluster.ipynb [hunk 4]
```diff
@@ -0,0 +1,865 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce58dc55-88d6-43f7-a18c-1974642ff0ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!terra resource resolve --name ws_files_autodelete_after_two_weeks || echo Be sure to run workspace_setup.ipynb first before this notebook."
```
Comment (deflaux) [inline dataproc/create_hail_cluster.ipynb]: nit: line is a bit too long, consider adding slash-newline so the bash code displays well on GitHub

File: dataproc/create_hail_cluster.ipynb [hunk 5]
```diff
@@ -0,0 +1,865 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a83c0ee2-39fc-47e8-babb-46949b5c05ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile init_tvc_hail_cluster.py\n",
+    "#!/opt/conda/default/bin/python3\n",
+    "\n",
+    "# This script is based on gs://hail-common/hailctl/dataproc/0.2.108/init_notebook.py with all JupyterLab configuration removed.\n",
```
Comment (deflaux) [inline dataproc/create_hail_cluster.ipynb]: nit: line is a bit too long, consider adding newline so the comment displays well on GitHub

File: dataproc/create_hail_cluster.ipynb [hunk 6]
```diff
@@ -0,0 +1,865 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5d3c60c-b616-45e7-b1be-c3cc8625d7fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud dataproc clusters describe {HAIL_CLUSTER_NAME} --region=us-central1 --format=\"yaml(config.endpointConfig.httpPorts)\""
```
Comment (deflaux) [inline dataproc/create_hail_cluster.ipynb]: nit: line is a bit too long, consider adding slash-newline so the bash code displays well on GitHub

**Review by deflaux — APPROVED** [review]
Nice updates to the markdown @amygdala !!! 

Just a few nits and also consider reverting back to the original language regarding need to run `workspace_setup.ipynb` once per workspace.

**Comment by amygdala** [comment]
thanks, Nicole!

**Pull request merged**